js/livecoin.js

```diff
@@ -86,6 +86,18 @@ module.exports = class livecoin extends Exchange {
                 'CRC': 'CryCash',
                 'XBT': 'Bricktox',
             },
+            'exceptions': {
+                '1': ExchangeError,
+                '10/11/12/20/30/101/102': AuthenticationError,
```

Comment:
I'd propose to split these into separate codes, that would help simplify the handler below somewhat. What do you think?

Reply:
Ok, it seems like the seemly clever idea blew up in my face when trying to transpile to php. Can you check the token I added to transpile.js?